packages/ethereum-contracts/contracts/superfluid/SuperToken.sol

```diff
@@ -688,6 +688,18 @@ contract SuperToken is
         _transferFrom(account, spender, recipient, amount);
     }
 
+    function operationSend(
+        address spender,
+        address recipient,
+        uint256 amount,
+        bytes memory userData
+    ) 
+        external override
+        onlyHost
+    {
+        _send(msg.sender, spender, recipient, amount, userData, "", userData.length != 0);
```

Comment:
we retain the `userData.length != 0` rule for `requireReceptionAck`

Reply:
should follow what send does

```
    function send(address recipient, uint256 amount, bytes calldata data) external override {
        _send(msg.sender, msg.sender, recipient, amount, data, "", true);
    }
```

Reply:
done